lib/protoc.dart

```diff
@@ -15,15 +15,15 @@ import 'src/dart_options.pb.dart';
 import 'const_generator.dart' show writeJsonConst;
```

Comment:
How so?

Our convention is to keep parts out of the root lib dir. It makes it much easier to figure out what's a library to import and what's a part.